src/main/java/lotto/domain/Lottos.java

```diff
@@ -1,20 +1,35 @@
 package lotto.domain;
 
-import java.util.Iterator;
-import java.util.List;
+import java.util.*;
 
-public class Lottos implements Iterable<Lotto>{
-    List<Lotto> lottoList;
+public class Lottos{
+    static List<Lotto> lottoList;
     public Lottos(List<Lotto> lottoList) {
         this.lottoList = lottoList;
     }
 
+    public static Map<Rank, Integer> calculateWinResult(Lottos lottos, Lotto winLotto, LottoNumber bonus) {
```

Comment:
```suggestion
    public static Map<Rank, Integer> calculateWinResult(Lotto winLotto, LottoNumber bonus) {
```
사용하지 않는 파라미터는 제거해주세요!

Reply:
앗ㅠㅜ 알려주셔서 감사합니다! 👍👍